CHANGELOG.rst

```diff
@@ -4,6 +4,25 @@ ibm.ibm\_zos\_core Release Notes
 
 .. contents:: Topics
 
+v1.9.3
+======
+
+Release Summary
+---------------
+
+Release Date: '2024-10-23'
```

Comment:
would this be 10-23, which was last week?

Reply:
updated it